### DISCUSSION TITLE
I need Github student bag for help with Twilio

### DISCUSSION THREAD

**Discussion by 1746705990** [community/community#55740] [GitHub Education]
### Hi everyone,

Because the previous discussion was not effectively replied to, I have to send it again!

It is worth noting that my Github student bag has been applied for, which is very gratifying.

But there is one item in Github's student bag that I cannot use, which is Twilio's rights. Specifically, when I registered for a Twilio account and prepared to try out the Twilio content Github brought me, my Twilio account was unreasonably banned. When I try to submit a work order and apply for Twilio to lift the ban, Twilio's staff always refuse without reason. This situation has been going on for a week, which makes me very uncomfortable.

Because I am preparing to learn how to use Python code to call interfaces to send and receive messages, there are also many cases on Github that are worth learning from. However, the blockade of my account by Twilio's staff directly invalidated my learning plan, and I had to postpone the plan, which made me very angry.

Because this responsibility belongs to Twilio, Github gave me the opportunity, but Twilio can intervene at will. I have to believe that this is a small trick Twilio did to evade responsibility, which made me very angry, especially when I had already communicated for a long time. As this belongs to the Github student package and I am still a college student, I have the right to use Twilio.

That's all. I hope to get it because my studies require Twilio's communication interface.
I have attached the email sent to me by Twilio in the previous discussion.

I have already stated in my email that I have obtained the Github student package, but the Twilio staff still suspended my account. I told him that I am a student and personal developer, but I am powerless. I think Twilio just doesn't want me to try Github's student bag, so slandering me as a scammer, damaging my reputation, and informing me that my account has fraudulent properties is really not convincing.

Finally, I hope @twilio-dx @twilio can help me because I believe the existence of this Github and the successful authentication of the student package can fully prove that my account does not have fraudulent properties. Thank you.

**Comment by samus-aran** [comment]
Hi @1746705990 

Thank you for contacting GitHub Education about the Twilio benefit. As of 19th June 2023, the Twilio benefit is no longer offered in the GitHub Student Developer Pack.

For students who have already redeemed your credit towards a new account, that credit will remain & you can contact Twilio's support team with any related questions [https://support.twilio.com/hc/en-us/articles/360048500694-Contacting-Twilio-Support]. Please note that not all offers are permanent and that, in some cases, the partner chooses to leave the Student Developer Pack.

We encourage you to explore Twilio's [trial account](https://support.twilio.com/hc/en-us/articles/223136107-How-does-Twilio-s-Free-Trial-work-), which includes a balance to get you started with Twilio's APIs.

For the most up-to-date list of partner offers, please visit this repo ([https://github.com/education/Student-Developer-Pack-Current-Partners-FAQ](https://urldefense.com/v3/__https://github.com/education/Student-Developer-Pack-Current-Partners-FAQ__;!!NCc8flgU!bx6wU7TIpsnj0KIwoVF4SQjbKROF1T6YX3zla5Lurz2SdjxrP33pA71fzU234-uGgHcOJ3TDq0fm7TzYf55pse_8$)) which is updated regularly.

**Comment by 1746705990** [comment]
@samus-aran 
Thank you for your response, but as I mentioned earlier, once I have registered my Twilio account, it will be locked out. Your answer may not match my question very well.
Although I have contacted Twilio support many times, the reason given is not to unlock and there is no reason, which is why I have the idea of seeking help in the community. I posted an extra post because, despite my previous post, no one has helped me for a long time, and I urgently need help.
